src/main/kotlin/org/jitsi/nlj/dtls/DtlsStack.kt

```diff
@@ -83,6 +83,20 @@ abstract class DtlsStack : DatagramTransport {
     val localFingerprintHashFunction: String
         get() = DtlsStack.getCertificate().localFingerprintHashFunction
 
+    /**
+     * The remote fingerprints sent to us over the signaling path.
+     */
+    var remoteFingerprints: Map<String, String> = HashMap()
+
+    /**
+     * Checks that a specific [Certificate] matches the remote fingerprints sent to us over the signaling path.
+     */
+    protected fun verifyAndValidateRemoteCertificate(certificate: Certificate){
+        DtlsUtils.verifyAndValidateCertificate(certificate, remoteFingerprints)
+        // The above throws an exception if the checks fail.
+        logger.debug("Fingerprints verified.")
```

Comment:
you can use `logger.cdebug { ... }` this has a built-in check to see if debug is enabled and, only if it is, does it execute the function to get the log text

Reply:
Any preference on whether I should use it when the argument is a single string literal?

Reply:
did we not wrap it with a check for `if (logger.isDebugEnabled())` in that case?  Honestly I've just taken to always using them

Reply:
i say we always use the conditional logs for now (cdebug, etc.) and we'll see if the hit is noticeable.  there are also loggers which are smarter about this, i think it's worth considering upgrading our logging libs at some point.